moveit_setup_assistant/src/widgets/setup_assistant_widget.cpp

```diff
@@ -131,9 +131,9 @@ SetupAssistantWidget::SetupAssistantWidget(QWidget* parent, boost::program_optio
   nav_name_list_ << "Robot Poses";
   nav_name_list_ << "End Effectors";
   nav_name_list_ << "Passive Joints";
```

Comment:
Reorder the nav bar as it makes more sense this was, as the simulation screen uses the controllers.

Reply:
love it